pkgs/by-name/co/cocom-tool-set/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+  bison,
+}:
+stdenv.mkDerivation {
+  pname = "cocom";
+  version = "0.996";
+
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/cocom/git";
+    sha256 = "sha256-utLafkznMC4LrZgF6vKehtIGMwNMwLP9M9Nwu/RyWio=";
+    rev = "64ee80224aa13f9944d439f3f90862ca76158705";
+  };
```

Comment:
Why not the [release tarball](https://sourceforge.net/projects/cocom/files/cocom%20sources/cocom-0.996/)?